Makefile

```diff
@@ -181,4 +182,7 @@ preflight:
 	$ npm install
 	$ bin/sync-all.sh
 
-.PHONY: all clean build pull docs livedocs build-docs lint run stop kill run-shell shell test test-image rebuild build-ci test-ci fresh-data djshell run-prod run-pocket run-pocket-prod build-prod test-cdn compile-requirements check-requirements install-local-python-deps install-local-docs-deps preflight
+clean-local-deps:
+	pip uninstall mdx_outline -y && pip freeze | xargs pip uninstall -y
```

Comment:
I forget, why does this need a special callout?

Reply:
`mdx_outline` is installed via a pinned tag from github rather than from PyPI and uninstalling didn't work `xargs` so it needed a special case IIRC